packages/teams-js/src/public/media.ts

```diff
@@ -597,6 +597,11 @@ export namespace media {
     assembleAttachment: AssembleAttachment[];
   }
 
+  /**
+   * Select an attachment using camera/gallery
+   *
+   * @returns  A promise resolved with an array of media data or rejected with an @see SdkError
```

Comment:
Thanks for turning this out quickly! Can you add `@param` for mediaInputs please?

Reply:
omg of course, sorry i'm still not used to the double documentation